packages/run-protocol/test/runStake/test-runStake.js

```diff
@@ -68,29 +69,13 @@ test.before(async t => {
  */
 const theBundles = t => /** @type { any } */ (t.context).bundles;
 
-export const setUpZoeForTest = async () => {
-  const { makeFar } = makeLoopback('zoeTest');
-
-  const { zoeService, feeMintAccess: nonFarFeeMintAccess } = makeZoeKit(
-    makeFakeVatAdmin(() => {}).admin,
-  );
-  /** @type {ERef<ZoeService>} */
-  const zoe = makeFar(zoeService);
-  const feeMintAccess = await makeFar(nonFarFeeMintAccess);
-  return {
-    zoe,
-    feeMintAccess,
-  };
-};
-harden(setUpZoeForTest);
-
 export const setupBootstrap = async (
   bundles,
   timer = buildManualTimer(console.log),
   zoe,
 ) => {
   if (!zoe) {
-    ({ zoe } = await setUpZoeForTest());
+    zoe = await setUpZoeForTest().zoe;
```

Comment:
does that mean the same thing? the former awaits the result of `setUpZoeForTest()` and then gets the `.zoe` property; the latter gets the `.zoe` property and then awaits it, no?

Reply:
Good catch. `produce.zoe.resolve(zoe)` now has a fully resolved `zoe` instead of an eref. Reading the types I think `EconomyBootstrapSpace` takes Erefs:
```
    ({ zoe } = setUpZoeForTest());
```

Reply:
I *changed* `setUpZoeForTest` to synchronously return a pair of promises. So the change is deliberate. I'm up for alternative naming, etc.  This is to reduce the awaits necessary in various places.

Reply:
ah. ok.